### DISCUSSION TITLE
What is the chat built on?

### DISCUSSION THREAD

**Discussion by cyrneko** [zed-industries/zed#14399] [Feature Requests]
As a new user I've quickly discovered the collaborative code editing features and am wondering, what is the chat functionality built on? Matrix? XMPP? Something else entirely?

**Comment by SomeoneToIgnore** [comment]
Something else entirely, it uses `collab` as everything else remote-related in Zed currently.
Here's an entry point to track around: https://github.com/zed-industries/zed/blob/89fbd6528f688cc98eaa4b3002f4b41da6c7b520/crates/proto/proto/zed.proto#L1447-L1454
